lumen/command/ai.py

```diff
@@ -0,0 +1,195 @@
+from __future__ import annotations
+
+import argparse
+import ast
+import os
+import sys
+
+import bokeh.command.util  # type: ignore
+
+from bokeh.application.handlers.code import CodeHandler  # type: ignore
+from bokeh.command.util import die  # type: ignore
+from panel.command import Serve, transform_cmds
+from panel.io.server import Application
+
+from ..config import config
+
+SOURCE_CODE = """
+import pathlib
+
+import lumen as lm
+import lumen.ai as lmai
+import panel as pn
+
+from lumen.sources.duckdb import DuckDBSource
+
+pn.extension("tabulator", "codeeditor", inline=False, template="fast")
+
+llm = lmai.llm.OpenAI()
```

Comment:
Would be nice to allow other providers here. We should try playing with Anthropic (Claude) and Google (Gemini) wrappers soon.

Reply:
Yeah, I created a separate issue https://github.com/holoviz/lumen/issues/604